Mergin/metadata.txt

```diff
@@ -162,7 +162,7 @@ changelog=2022.3.2
 ; tags are in comma separated value format, spaces are allowed
 ; tags=
 
-homepage=https://public.cloudmergin.com
+homepage=https://app.merginmaps.com/
```

Comment:
I think we should rather point to canonical URL with landing page https://merginmaps.com

and in readme we should probably use that one as well except the case where we really want to point to mergin app (like working default URL) - simply because default root for app.merginmaps would display login dialog